src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        File file = new File(fromFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] parts = line.split(",");
+                if (parts.length == 2) {
+                    String operationName = parts[0].trim();
+                    int amount = Integer.parseInt(parts[1].trim());
+
+                    if (operationName.equals("supply")) {
+                        supply += amount;
+                    } else if (operationName.equals("buy")) {
+                        buy += amount;
+                    }
+                }
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        int result = supply - buy;
 
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write("supply," + supply);
```

Comment:
Use StringBuilder to generate report
don't use concatenation to avoid creating new String objects